### PR TITLE
PricingModel Test Case

### PR DESCRIPTION
**Pull request by jorgeorpinel**

### CODE REVIEW

File: test/Controllers/PricingModelsTest.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+     * @param  string $type Should be 'create' or 'update'.
+     * @return  CreatePricingModelBody or UpdatePricingModelByIdBody object with unique data to send to API
+     */
+    private function pricingModelBody($type) {
+        switch($type) {
+            case 'create':
+                return new CreatePricingModelBody(
+                    "name",
+                    "Activity", // type
+                    "USD", // currency XXX This isn't having effect.
```
Comment (alexagranov) [inline test/Controllers/PricingModelsTest.php]: Let's take out the comment please.  We can keep these kind of notes internally.

**Comment by alexagranov** [comment]
please resolve conflicts

File: test/Controllers/PricingModelsTest.php (hunk 1 above)
Comment (jorgeorpinel) [inline test/Controllers/PricingModelsTest.php]: Done. And yes I mostly add codetags so we notice them in the PRs! But I guess it's not the best system, I can just note it here directly without he code comments.

**Comment by jorgeorpinel** [comment]
Ready, @alexagranov

**Pull request merged**